README.md

```diff
@@ -24,7 +24,9 @@ bower install planout
 
 PlanOut.js provides an implementation of all PlanOut features (including the
 experiment class, interpreter, and namespaces). The underlying randomization ops in the JavaScript implementation return different
-results for efficiency reasons. If you are using PlanOut cross-platform and want to enable compatibility mode then you can enable it by calling ```ExperimentSetup.toggleCompatibleHash(true)```.
+results for efficiency reasons. If you are using PlanOut cross-platform and want to enable compatibility mode then you can enable it by utilizing the ```planout_core_compatible.js``` distribution bundle instead of the default ```planout.js``` bundle.
```

Comment:
How does one actually utilize the bundle? is the only way to bring in the compat file manually since afaik we can only publish one under the npm package and I don't think we should create a separate npm package for the compat one since that's a bit of a slippery slope